### PR TITLE
Animation position utils should take scrollbar width into account (T939748)

### PR DESCRIPTION
**Pull request by ksercs**
…

### CODE REVIEW

File: testing/tests/DevExpress.animation/position.tests.js [hunk 1]
```diff
@@ -955,6 +955,34 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         }
     });
 
+    QUnit.test('position should return window.height() if window.outerHeight == window.innerHeight (T939748)', function(assert) {
+        const isPhone = devices.real().deviceType === 'phone';
+        if(isPhone || browser.safari) {
+            assert.expect(0);
```
Comment (DokaRus) [inline testing/tests/DevExpress.animation/position.tests.js]:
```suggestion
            assert.ok(true, 'tell me why');
```

File: testing/tests/DevExpress.animation/position.tests.js [hunk 2]
```diff
@@ -955,6 +955,34 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         }
     });
 
+    QUnit.test('position should return window.height() if window.outerHeight == window.innerHeight (T939748)', function(assert) {
+        const isPhone = devices.real().deviceType === 'phone';
```
Comment (DokaRus) [inline testing/tests/DevExpress.animation/position.tests.js:960]: Is it actual only for phones? Are the tablets OK?

File: testing/tests/DevExpress.animation/position.tests.js (hunk 1 above)
Comment (ksercs) [reply testing/tests/DevExpress.animation/position.tests.js]: fixed

File: testing/tests/DevExpress.animation/position.tests.js (hunk 2 above)
Comment (ksercs) [reply testing/tests/DevExpress.animation/position.tests.js:960]: There is the same behavior on tablets as on desktops

**Review by DokaRus — DISMISSED** [review]

**Review by San4es — DISMISSED** [review]

**Review by alexander-kotov-dx — DISMISSED** [review]

File: testing/tests/DevExpress.animation/position.tests.js [hunk 3]
```diff
@@ -955,6 +956,64 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
...
+        }
+
+        const $what = $('#what').height(300);
+        const initialInnerHeight = window.innerHeight;
+        const initialOuterHeight = window.outerHeight;
+
+        const heightStub = sinon.stub(renderer.fn, 'height').returns(1000);
+
+        try {
+            window.innerHeight = 500;
```
Comment (nightskylark) [inline testing/tests/DevExpress.animation/position.tests.js:973]: I think it's better to use a mock for window instead of modifying a real one.

Comment (ksercs) [reply testing/tests/DevExpress.animation/position.tests.js:973]: How can we do it? `innerHeight` is property, not method, so we can not use `sinon.stub`

**Review by San4es — DISMISSED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**